Added cluster-health check instructions (#18958)
kubernetes/website: content/en/docs/tasks/debug-application-cluster/debug-cluster.md

And verify that all of the nodes you expect to see are present and that they are all in the `Ready` state.

> To get detailed information about the overall health of your cluster, you can run:.

Reviewer: ```suggestion
To get detailed information about the overall health of your cluster, you can run:
```

Author: Done @sftim

And verify that all of the nodes you expect to see are present and that they are all in the `Ready` state.

> Also, you can run below command to get detailed information of overall cluster health.

Reviewer: ```suggestion
To get detailed information about the overall health of your cluster, you can run:
```
?

@rkatti, what do you think?

Author: @sftim Looks good

```shell

> 

Reviewer: This blank line doesn't seem to be needed here.

Author: Agree @sftim

Reviewer 2: @rikatz you forgot to address this review comment!

Reviewer 3: I think you mean @rkatti :)

Author: @sftim @makoscafee have addressed the comments. Pls, review.
